docs/source/examples/Analyzing the Parker Solar Probe flybys.ipynb

```diff
@@ -781,18 +781,17 @@
    "outputs": [],
    "source": [
     "from astropy.coordinates import CartesianRepresentation, CartesianDifferential\n",
-    "from poliastro.frames import Planes, get_frame\n",
-    "\n",
+    "from poliastro.frames import Planes\n",
     "\n",
     "def change_plane(ss_orig, plane):\n",
     "    \"\"\"Changes the plane of the Orbit.\n",
     "\n",
     "    \"\"\"\n",
-    "    ss_orig_rv = ss_orig.frame.realize_frame(\n",
+    "    ss_orig_rv = ss_orig.get_frame().realize_frame(\n",
     "        ss_orig.represent_as(CartesianRepresentation, CartesianDifferential)\n",
     "    )\n",
     "\n",
-    "    dest_frame = get_frame(ss_orig.attractor, plane, obstime=ss_orig.epoch)\n",
+    "    dest_frame = ss_orig.get_frame()\n",
```

Comment:
I think we should still be using the `get_frame` function here, otherwise the input `plane` is not used!